velox/docs/functions/spark/datetime.rst

```diff
@@ -133,6 +133,19 @@ These functions support TIMESTAMP and DATE input types.
     ``day`` need to be from 1 to 31, and matches the number of days in each month.
     days of ``year-month-day - 1970-01-01`` need to be in the range of INTEGER type.
 
+.. spark:function:: make_ym_interval([years[, months]]) -> interval year to month
+
+    Make year-month interval from ``years`` and ``months`` fields.
```

Comment:
Would you clarify what happens when months is > 12 or less than -12? Is 'months' zero-based or 1-based? Would be nice to clarify as well.

Reply:
Month > 12 or < -12 are valid cases. e.g. `make_ym_interval(1, 100); -- 9-4`. Added this example to document and UT.